models/ActionCancel.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace app\models;
+
+class ActionCancel extends Action
+{
+    public function getName()
+    {
+        return "Отменить";
+    }
+
+    public function getInnerName()
+    {
+        return "cancel";
+    }
+
+    public function checkUserRole($userId, $customerId, $executorId)
+    {
+        return ($userId == $customerId) ? true : false;
```

Comment:
здесь и в других методах достаточно вот такого выражения, без использования тернарного оператора, он избыточен:
```suggestion
        return $userId == $customerId;
```